textdb/textdb-angular-gui/app/sidebar/side-bar.component.html

```diff
@@ -23,6 +23,18 @@ <h3 *ngIf="attributes.length !== 0" class="sidebar-brand">{{operator}}</h3>
                   <option *ngFor="let agg of aggregationList" [value] = "agg">{{agg}}</option>
                 </select>
 
+                <div *ngIf="attribute === 'attributes'">
```

Comment:
Actually some operators have `attribute` property, which is only one attribute, instead of multiple attributes in the `attributes` property